src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -73,12 +77,21 @@ public void testDoPost_BadUsername() throws IOException, ServletException {
   @Test
   public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("username")).thenReturn("existing username");
+    Mockito.when(mockRequest.getParameter("password")).thenReturn("existing password");
```

Comment:
you probably don't need this...it doesn't hurt to leave it here either...can ignore